shell/platform/darwin/macos/framework/Source/FlutterPlatformViewController.mm

```diff
@@ -116,11 +116,43 @@ - (void)handleMethodCall:(nonnull FlutterMethodCall*)call result:(nonnull Flutte
     NSNumber* arg = [call arguments];
     int64_t viewId = [arg longLongValue];
     [self onDisposeWithViewID:viewId result:result];
+  } else if ([[call method] isEqualToString:@"acceptGesture"]) {
+    [self handleAcceptGesture:call result:result];
+  } else if ([[call method] isEqualToString:@"rejectGesture"]) {
+    [self handleRejectGesture:call result:result];
   } else {
     result(FlutterMethodNotImplemented);
   }
 }
 
+- (void)handleAcceptGesture:(FlutterMethodCall*)call result:(FlutterResult)result {
+  NSDictionary<NSString*, id>* args = [call arguments];
+  int64_t viewId = [args[@"id"] longLongValue];
```

Comment:
Should this check that the arg exists and is numeric to avoid crashes on malformed messages?

Reply:
In Objective-C, messages to `nil` are evaluated as `nil`/`0`, so in the case where no key `id` exists, `viewId` will be set to 0. PlatformView IDs start at 1, so this will return an error rather than crash. 

That said, I agree an explicit check would significantly help readability.

Reply:
Added assertions since lack of an `id` param would be a programming error on our part (in the framework).